index.js

```diff
@@ -184,7 +185,13 @@ function resolveWebpackPath(partial, filename, directory, webpackConfig) {
 
   try {
     var loadedConfig = require(webpackConfig);
-    var resolver = webpackResolve.create.sync(loadedConfig.resolve || {});
+    var resolveConfig = objectAssign({}, loadedConfig.resolve);
+
+    if (resolveConfig.root) {
+      resolveConfig.modules = [].concat(resolveConfig.root);
```

Comment:
The question is if we maybe should support http://webpack.github.io/docs/configuration.html#resolve-modulesdirectories too? I suppose things could go wrong if a user has set a path in `root` but not `node_modules`. It will then fail to resolve paths in `node_modules`. Maybe we should always ensure `node_modules` is in `resolveConfig.modules`?